### PR TITLE
fix: bash deps

### LINKED ISSUE #750: Mybash deps check is bugged
## Describe the bug
`mybash-setup.sh` uses `command_exists "bash" "bash-completion" ...` as if it fails if one of the dependencies does not exist, but this won't work as written. Also, `bash-completion` and `fontconfig` aren't executables, so the check would always fail if it did work as intended...

## To Reproduce
1. run the bash prompt script
2. install fails if deps aren't installed, most likely to happen on fresh distro installs

## Checklist
- [x] I checked for duplicate issues.
- [x] I checked already existing discussions.
- [x] This issue is not included in the roadmap.
- [x] This issue is present on both stable and development branches.

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
- bash-completion and fontconfig are not executables
- This PR depends on #762 

## Testing
- Works as expected

## Issues / other PRs related

- Resolves #750 

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]
~~close this~~

**Comment by nnyyxxxx** [comment]
wait

File: core/tabs/applications-setup/mybash-setup.sh [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
+    if [ ! -f "/usr/share/bash-completion/bash_completion" ] || ! command_exists bash tar bat tree unzip fc-list git; then
+        printf "%b\n" "${YELLOW}Installing Bash...${RC}"
+        case "$PACKAGER" in
+            pacman)
+                "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm bash bash-completion tar bat tree unzip fontconfig git
+                ;;
+            *)
+                "$ESCALATION_TOOL" "$PACKAGER" install -y bash bash-completion tar bat tree unzip fontconfig git
+                ;;
+        esac
```
Comment (nnyyxxxx) [inline core/tabs/applications-setup/mybash-setup.sh:17]:
```suggestion
    if ! command_exists bash tar bat tree unzip git; then
    printf "%b\n" "${YELLOW}Installing Bash...${RC}"
    case "$PACKAGER" in
        pacman)
            "$ESCALATION_TOOL" "$PACKAGER" -S --needed --noconfirm bash bash-completion tar bat tree unzip fontconfig git
            ;;
        *)
            "$ESCALATION_TOOL" "$PACKAGER" install -y bash bash-completion tar bat tree unzip fontconfig git
            ;;
    esac
```

just remove them from command_exists

Comment (nnyyxxxx) [reply core/tabs/applications-setup/mybash-setup.sh:17]: we shouldnt add unnecessary overhead, this is a much simpler way

Comment (jeevithakannan2) [reply core/tabs/applications-setup/mybash-setup.sh:17]: Removing them is incorrect. What if the user already has every other deps installed but not bash-completion ?

Comment (jeevithakannan2) [reply core/tabs/applications-setup/mybash-setup.sh:17]: Checking if a file exists does not create extra overhead 🤷🏻‍♂️

Comment (nnyyxxxx) [reply core/tabs/applications-setup/mybash-setup.sh:17]: mark this as resolved

**Review by nnyyxxxx — APPROVED** [review]

**Comment by nnyyxxxx** [comment]
tested and works :+1:

**Review by cartercanedy — APPROVED** [review]

**Review by adamperkowski — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**